tesseract_scene_graph/include/tesseract_scene_graph/kinematics_information.h

```diff
@@ -72,6 +72,8 @@ struct OPWKinematicParameters
 
 struct ROPKinematicParameters
 {
+  /** @brief The name of the solver. If empty it will use the default solver name */
+  std::string name;
```

Comment:
Should this be solver_name?

Reply:
Yea that is probably best. I went back and forth but solver_name is more explicit and probably better.

Reply:
Renamed.